R/ari_spin.R

```diff
@@ -130,8 +131,8 @@ ari_spin <- function(images, paragraphs,
     # coqui+ari doesn't work with mp3
     if (service == "coqui") {
       args$output_format <- "wav"
+      cli::cli_alert_info("Coqui TTS does not support MP3 format and will produce a WAV audio output.")
```

Comment:
Was this already a package that was dependency in here? Also how is this different than using `warning()` or `message()`? I've never heard of this package so I'm just curious.

Reply:
I need to add cli as a dependency in the docs! Also, this isn't much different from a `message()` except it makes prettier messages, like https://github.com/r-lib/cli#short-alert-messages